test/sequential/test-benchmark-http.js

```diff
@@ -20,6 +20,9 @@ const path = require('path');
 
 const runjs = path.join(__dirname, '..', '..', 'benchmark', 'run.js');
 
+const env = process.env;
```

Comment:
I find this less readable, so I'd be slightly -1 on this unless there's a benefit. Could you explain what you mean by code hygiene?